Dockerfile

```
@@ -0,0 +1,49 @@
+
+RUN mkdir -p build
+
+# Configure and build the project
+RUN cd build && \
+    cmake .. && \
+    make -j$(nproc)
+
+RUN mkdir -p artifacts/linux
+
+# Create a run script that will execute all examples
+RUN echo '#!/bin/bash' > run_examples.sh && \
```

Hadolint (2.12.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)